Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml

```
@@ -411,6 +446,240 @@ script:
+      type: Number
+    - contextPath: ASM.Incident.modification_time
+      description: 'Modification timestamp'
+      type: Date
+    - contextPath: ASM.Incident.notes
+      description: 'Incident notes'
+      type: Unknown
+    - contextPath: ASM.Incident.original_tags
+      description: 'Tags on the incident at creation time'
+      type: Unknown
+    - contextPath: ASM.Incident.resolve_comment
+      description: 'Resolution comment. This is optional'
+      type: Unknown
+    - contextPath: ASM.Incident.resolved_timestamp
+      description: 'Resolution timestamp'
+      type: Unknown
+    - contextPath: ASM.Incident.severity
+      description: 'Severity of the incident'
+      type: String
+    - contextPath: ASM.Incident.starred
+      description: 'Whether the incident has been starred'
+      type: Boolean
+    - contextPath: ASM.Incident.status
+      description: 'Status of the incident'
+      type: String
+    - contextPath: ASM.Incident.tags
+      description: 'Tags on the incident'
+      type: String
+    - contextPath: ASM.Incident.xdr_url
+      description: 'Link to navigate to the incident'
+      type: String
+    - contextPath: ASM.Incident.xpanse_risk_score
+      description: 'Risk score of the incident'
+      type: Unknown
+    description: 'Fetches ASM incidents that match provided filters. Incidents are
+      an aggregation of related alerts. Note: Incident IDs may also be references
+      as "Case IDs'' elsewhere in the API.'
+  - name: asm-update-incident
+    arguments:
+    - name: incident_id
+      required: true
+      description: ID of the incident to modify
+    - name: alert_id
+      description: Used for scoping updates such as comments to the alert level
+    - name: assigned_user_mail
+      description: Email address of the user to assign incident to. This user must
+        exist within your Expander instance
+    - name: manual_severity
+      description: Administrator-defined severity for the incident
+    - name: status
+      auto: PREDEFINED
+      predefined:
+      - new
+      - under_investigation
+      - resolved_-_no_longer_observed
+      - resolved_-_no_risk
+      - resolved_-_risk_accepted
+      - resolved_-_contested_asset
+      - resolved_-_remediated_automatically
+      - resolved
+      description: Incident status
+    - name: resolve_comment
+      description: Optional resolution comment when resolving the incident
+    - name: comment
+      description: A comment to add to the incident. If an alert_id is supplied it
+        will be prefixed to the comment.
+    outputs:
+    - contextPath: ASM.IncidentUpdate
+      description: Whether the incident update was successful
+    description: Updates a given incident. Can be used to modify the status, severity,
+      assignee, or add comments.
+  - name: asm-update-alerts
+    arguments:
+    - name: alert_id_list
+      description: Comma separated list of integers of the alert ID
```

```suggestion
      description: Comma-separated list of integers of the alert ID.
```
Perhaps change this to 
Comma-separated list alert IDs.